README.md

```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
 ## Installation
 
 ```bash
-pip install git+https://github.com/iMicknl/python-tahoma-api.git@master#tahoma_api
+pip install python-tahoma-api
+```
+
+## Getting started
+
+```python
+import asyncio
+import time
+
+from tahoma_api.client import TahomaClient
+
+USERNAME = ""
+PASSWORD = ""
+
+async def main() -> None:
+    async with TahomaClient(USERNAME, PASSWORD) as client:
+        try:
+            await client.login()
+        except Exception as exception:  # pylint: disable=broad-except
+            print(exception)
+            return await client.close()
```

Comment:
You can  just do `return`, the `with` block will close the session automatically.